python/social_bot/envs/embodied_teacher.py

```diff
@@ -0,0 +1,265 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A simple enviroment for an agent play on the ground
+"""
+import os
+import time
+import math
+import numpy as np
+import random
+import json
+import PIL
+import gin
+import gym
+from gym import spaces
+from absl import logging
+from abc import abstractmethod
+from collections import OrderedDict
```

Comment:
Not all the imports are used in this file.

Reply:
removed.